packages/@lwc/template-compiler/typings/parse5.d.ts

```diff
@@ -7,13 +7,6 @@
 import 'parse5-with-errors';
```

Comment:
_for my knowledge:_ Why is this import statement needed here?

Reply:
This allows bringing the `parse5-with-errors` types and interfaces into the scope. If you remove the import statement at the top you won't have access to the `AST.Default.DocumentFragment` interface.